client/my-sites/stats/stats-countries/index.jsx

```diff
@@ -1,72 +1,71 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+
+import React from 'react';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var toggle = require( '../mixin-toggle' ),
-	Geochart = require( '../geochart' ),
-	StatsList = require( '../stats-list' ),
-	observe = require( 'lib/mixins/data-observe' ),
-	skeleton = require( '../mixin-skeleton' ),
-	DownloadCsv = require( '../stats-download-csv' ),
-	DatePicker = require( '../stats-date-picker' ),
-	ErrorPanel = require( '../stats-error' ),
-	Card = require( 'components/card' ),
-	StatsModulePlaceholder = require( '../stats-module/placeholder' ),
-	StatsListLegend = require( '../stats-list/legend' ),
-	Gridicon = require( 'components/gridicon' );
-
-module.exports = React.createClass( {
+import toggle from '../mixin-toggle';
+import Geochart from '../geochart';
+import StatsList from '../stats-list';
+import observe from 'lib/mixins/data-observe';
+import DownloadCsv from '../stats-download-csv';
+import DatePicker from '../stats-date-picker';
+import ErrorPanel from '../stats-error';
+import Card from 'components/card';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import StatsListLegend from '../stats-list/legend';
+import Gridicon from 'components/gridicon';
+import SectionHeader from 'components/section-header';
+
+export default React.createClass( {
 	displayName: 'StatCountries',
 
-	mixins: [ toggle( 'Countries' ), skeleton( 'data' ), observe( 'dataList' ) ],
+	mixins: [ toggle( 'Countries' ), observe( 'dataList' ) ],
```

Comment:
hmmm I thought this was removed, but we should be able to rip out `toggle( 'Countries' )` and the `import` above as well.